p4-16/spec/P4-16-spec.mdk

```diff
@@ -6388,19 +6388,68 @@ annotations
 annotation
     : '@' name
     | '@' name '(' expressionList ')'
+    | '@' name '(' kvList ')'
+    ;
+...    
+kvPair
```

Comment:
Same here, I hope we can use keyValuePair instead of kvPair. It is more consistent with the rest of the BNF.

Reply:
I'm happy to make that change, thanks for proposing.

Reply:
Done